test/topology/test_tablets.py

```diff
@@ -47,9 +50,7 @@ async def test_tablet_metadata_propagates_with_schema_changes_in_snapshot_mode(m
         assert r.c == 1
 
     await manager.server_start(s0, wait_others=2)
-
-    manager.driver_close()
-    await manager.driver_connect(server=servers[0])
+    await wait_for_cql_and_get_hosts(manager.cql, [servers[0]], time.time() + 60)
```

Comment:
I see you're still using `execution_profile='whitelist'` below; does that still make sense, given that you no longer specify to connect to a single server (removed the `driver_connect` call) (now the driver will use any node)?

Reply:
It still makes sense, but I should bring back the `await manager.driver_connect(server=servers[0])` here. Will fix in v2.